libdnf/dnf-swdb-db-sql.h

```diff
@@ -136,18 +136,14 @@
     "   tw_id INTEGER PRIMARY KEY," \
     "   t_id INTEGER REFERENCES trans(t_id)," \
     "   p_id INTEGER REFERENCES package(p_id));" \
-    "CREATE INDEX nevra ON PACKAGE(" \
-    "   name || '-' || epoch || ':' || version || '-' || release || '.' || arch);" \
-    "CREATE INDEX nvra on PACKAGE(" \
-    "   name || '-' || version || '-' || release || '.' || arch);" \
+    "CREATE INDEX name_index ON PACKAGE(name);" \
```

Comment:
I was wondering if name_index is sufficient (why not to add more indexes?) but then I realized that name is a mostly unique identifier. Only multilib or installonly packages break this rule and we're talking about making additional scan on up to 3 packages. -> I'm definitely fine with this as it is.